endpoints/api/superuser.py

```diff
@@ -237,6 +237,26 @@ class SuperUserUserQuotaList(ApiResource):
         },
     }
 
+    @require_fresh_login
+    @verify_not_prod
+    @nickname(["listUserQuotaSuperUser", "listOrganizationQuotaSuperUser"])
```

Comment:
This is just listing users', not orgs'.

Reply:
Added `user.get_user_or_org(namespace)` function [here](https://github.com/quay/quay/pull/1334/files#diff-e1fe16b0a62b4ced394eb81261f3f60d5086988f3e7be21489f6df00bbcd6d4eR248) to support fetching orgs quota.